clusterloader2/docs/experiments.md

```diff
@@ -0,0 +1,44 @@
+# clusterloader2 - experiment rollout
+
+In this doc any change to the behavior of clusterloader2 is referred as an "experiment".
+
+## Motivation
+
+clusterloader2 is a tool used a by all scalability (performance) tests. Tests compile clusterloader2 at [HEAD](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L121). Introducing breaking changes to clusterloader2 will stop tests from passing at all. While for smaller and faster tests, this is not a huge costs (unless this happens on the weekend, see https://github.com/kubernetes/perf-tests/pull/586), this can be costly in rarely run, expensive tests (e.g. [ci-kubernetes-e2e-gce-scale-performance](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L44)).
```

Comment:
It can be also costly if we break k/k presubmit, as it blocks all k/k PRs from being merged.

Reply:
It's mentioned in point 2 under "general principles", that we don't want to break k/k presubmits. I'll be more explicit in this section about it too.